scripts/build.sh

```diff
@@ -15,6 +15,7 @@ case "$*" in
 	;;
 esac
 
+TARGET=""
```

Comment:
This is the actual fix.

Reply:
TARGET is set and then that variable is reused in here for something else. So this resets.